README-Oisin.md

```diff
@@ -0,0 +1,28 @@
+I managed to set-up the website test page and a chitter feed 'page'. Made some mistakes in setting up the test database, and then misnaming the 'views' folder as 'view' also wasted a lot of time!
+
+I was planning as I went along, so my plan is less thorough than other weeks. Below, you can see how I was identifying the pages/assets needed for the first 2 user stories.
+
+<!-- As a Maker
+So that I can see what people are doing
+I want to see all the messages (peeps)
+in a browser -->
+
+Who? - Maker
+Class - Chitter
+
+What is needed to achieve this?
+A database of peeps 
+A page to display all the peeps in a list
+
+<!-- As a Maker
+So that I can let people know what I am doing  
+I want to post a message (peep) to chitter -->
+
+Who? - Maker
+Class - peep
+
+What is needed to achieve this? 
+A page to submit a peep
+A form to save a peep to the database
+
+
```

Comment:
What do you think about adding a section for cloning/running instructions? You could include how to clone your code from GitHub, how to run your app (`rackup`) and how to create the database locally on their machine.